index.html

```diff
@@ -83,5 +83,270 @@ <h1 class="text-dark hero-title">
         </button>
       </div>
     </section>
+
+    <section class="work-section">
+      <header>
+        <h1 class="section-header text-center">My Recent Works</h1>
+        <span class="underline"></span>
```

Comment:
- [ ] Please consider using other heading elements(`h2`, `h3` ... ) since it is recommended to use the `h1` tag once in the HTML document. :pray: